modules/gitops-release-notes-1-12-1.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="release-notes-for-gitops-1-12-1_{context}"]
+= Release Notes for {gitops-title} 1.12.1
+
+{gitops-title} 1.12.1 is now available on {OCP} 4.12, 4.13, 4.14, and 4.15.
+
+[id="errata-updates-1-12-1_{context}"]
+== Errata updates
+
+[id="rhsa-2024:1753-gitops-1-12-1-security-update-advisory_{context}"]
+=== RHSA-2024:1753 - {gitops-title} 1.12.1 security update advisory
+
+Issued: 2024-04-08
```

Comment:
I'm seeing the issue date in https://access.redhat.com/errata/RHSA-2024:1753 as 4/10:
```suggestion
Issued: 2024-04-10
```